app/models/user.rb

```diff
@@ -487,7 +487,7 @@ def title
   def number_visits_per_day
     return user.sign_in_count.round(1) if user.created_at.today?
 
-    (user.sign_in_count.to_f / (Time.zone.today - user.created_at.to_date).to_i).round(1)
+    (user.sign_in_count.to_f / (Time.zone.today - user.created_at.to_date).to_i).round
```

Comment:
additional decimal places don't bring meaningful information for this stat, round to the closest integer